ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -0,0 +1,433 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * expect
+*)
+
```

Comment:
I did not review these error messages for quality. Just completeness. Please add a CR to the top of this file saying we have to review for quality.